agent/recorder-agent.c

```diff
@@ -351,13 +347,17 @@ get_records (gchar * recording_dir, gchar * arg_edge_id, gchar * arg_record_id,
   GDir *subdir = NULL;
   GError *error = NULL;
   const gchar *filename = NULL;
+  gchar *edge_id_tmp = NULL;
   gchar *edge_id = NULL;
   gchar *record_id = NULL;
   gchar *file_id = NULL;
   gchar *recording_edge_dir = NULL;
   gboolean records_found = FALSE;
   GStatBuf st;
   gchar *filename_full = NULL;
+  gint64 file_start = 0;
+  gint64 file_end = 0;
```

Comment:
You  can limit the scope of these variables to the for loop that iterates `file_list`.

Reply:
Thanks